rollup.config.ts

```diff
@@ -0,0 +1,44 @@
+import replace from '@rollup/plugin-replace'
+import { defineConfig } from 'rollup'
+import rename from 'rollup-plugin-rename'
+import typescript from 'rollup-plugin-ts'
+
+export default defineConfig({
+  input: './src/index.ts',
+  plugins: [
+    replace({
+      preventAssignment: true,
+      values: { '.ts': '.js' },
+    }),
+    replace({
+      preventAssignment: false,
+      values: { '// %isomorphic-ws%': 'import WebSocket from "isomorphic-ws"' },
+      delimiters: ['', ''],
+    }),
+    // fixes bundled tslib becoming cjs because it's inside a node_modules dir
+    rename({ map: name => name.replace(/node_modules/, 'external') }),
```

Comment:
That is strange! Will have a look into it.

Reply:
That's just the default behaviour, each directory inside `node_modules` will be treated as a package, and by default all packages are commonjs if not defined by `package.json`